experimental/dds/tree2/src/feature-libraries/modular-schema/typedSchema/typeUtils.ts

```diff
@@ -15,7 +15,7 @@ import { Named } from "../../../core";
  * @remarks
  * This is useful in generic code when it is impractical (or messy)
  * to to convince the compiler that a generic type `TInput` will extend `TAssumeToBe`.
- * In these cases `TInput` can be replaced with `Assume<TInput, TAssumeToBe>` to allow complication of the generic code.
```

Comment:
🤣

Reply:
It is strange that this shows up here, because this comes from PR #15307.  Feel like a github bug to me.